schemas/bedrock-kms-http.js

```diff
@@ -0,0 +1,192 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const controller = {
+  title: 'controller',
+  type: 'string'
+};
+
+const referenceId = {
+  title: 'referenceId',
+  type: 'string'
+};
+
+const delegator = {
+  anyOf: [{
+    type: 'string'
+  }, {
+    type: 'array',
+    minItems: 1,
+    items: {type: 'string'}
+  }]
+};
+const invoker = {
+  anyOf: [{
+    type: 'string'
+  }, {
+    type: 'array',
+    minItems: 1,
+    items: {type: 'string'}
+  }]
+};
+
+const postKeystore = {
+  title: 'postKeystore',
+  type: 'object',
+  additionalProperties: false,
+  required: ['sequence', 'controller'],
+  properties: {
+    controller,
+    referenceId,
+    delegator,
+    invoker,
+    sequence: {
+      title: 'sequence',
+      type: 'number',
+      minimum: 0,
+      maximum: Number.MAX_SAFE_INTEGER - 1
+    }
+  }
+};
+
+const findKeystore = {
+  title: 'findKeystore',
+  type: 'object',
+  additionalProperties: false,
+  required: ['controller', 'referenceId'],
+  properties: {
+    controller,
+    referenceId
+  }
+};
+
+const zcap = {
```

Comment:
A note here that we do not want to maintain a complex schema like the one for zcap in multiple places. The solution that comes to my mind is to have a `bedrock-zcap` module that primarily provides this schema to bedrock so that it can be used in all the places (e.g. bedrock-kms-http, bedrock-edv-storage).

I think we should discuss this soon and then make an issue to implement that separate from this PR.

Reply:
Agreed, that would simplify adding validators while maintaining accuracy. +1 to moving it to it's own module.

Reply:
Created issue to track this.